spec/features/publish/courses/editing_application_status_spec.rb

```diff
@@ -21,10 +31,24 @@ def and_i_should_be_back_on_the_course_page
                                                                                      course.course_code))
   end
 
+  def and_the_course_is_open
+    course.reload
+    expect(course.application_status_open?).to be true
```

Comment:
this is me stretching but are we able to get `expect(course). to be_application_status_open`

Reply:
Thats cool